week3/Program.cs

```diff
@@ -0,0 +1,135 @@
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+app.MapGet("/", () => "Hello World!");
+
+app.MapGet("/class-exer1", () =>
+{
+    var temperature = new Temperature(300);
+    Console.WriteLine($"{temperature.Celsius} Celsius is {temperature.ConvertToFahrenheit()} Fahrenheit");
+});
+
+app.MapGet("/class-exer2", () =>
+{
+    var amount = 100;
+    var exchangeRate = new ExchangeRate("EUR", "DKK");
+    exchangeRate._rate = 7.5m;

```

Comment:
this is a field and it is usually marked as private and is not accessible outside of the class. 
If you want to do this you should change it to a property 😊